subjects/zappy/README.md

```diff
@@ -0,0 +1,368 @@
+## zappy
+
+### Objectives
+
+You have to create Zappy, which is an entirely automatic game where some AI's play amongst themselves. It is about creating families, surviving by eating food and collecting stones in order to level up. Each level has different requirements for a ritual, which makes the adventure more and more difficult for the AI players, which play on their own, without any help form their creator. The game is over once six members of one family reach the highest possible level (8).
+
+### Game presentation
+
+### The Game Parts
+
+Here are the different parts of the game:
+
+- The `server` contains the field, the resources, and manages the logic and timing of the game.
+
+- One or many `clients` will connect to the server and each will “pilot” one player. The players are spread over teams.
+
+- A `graphic client` that connects to the server and displays the field and what happens there.
+
+#### Geography
+
+- The game consists in managing an entire world and its population. That world, "Trantor" is geographically made of plains that have no height, no crater, no valley or mountains.
```

Comment:
- the name of the world `Trantor`, should be changed
- the part stating `no heights, no crater...` should be changed (exactly the same of subjects already existing), maybe we can just say `there won't be any obstacle`

Reply:
done